karapace/schema_registry_apis.py

```diff
@@ -1104,14 +1104,14 @@ async def subjects_schema_post(
                 schema_type=schema_type, schema_str=schema_str, references=references, dependencies=new_schema_dependencies
             )
         except InvalidSchema:
-            self.log.exception("No proper parser found")
+            self.log.warning("Invalid schema: %r", schema_str, exc_info=True)
```

Comment:
suggestion: I noticed this after approving ... But passing `exc_info=True` here will put a traceback in the logs. For an expected error like this I think that's too verbose.